.gitignore

```diff
@@ -111,3 +111,7 @@ openapi.yml
 
 #Eclipse project
 .project
+docker-compose.yml
```

Comment:
Are the `.gitignore` updates intentional? Files listed seem random-ish>

Reply:
Mijn fout it was not supposed to be pushed

Reply:
`.gitignore`has is now identical to master